process_report/tests/unit_tests.py

```diff
@@ -234,29 +234,19 @@ def test_get_pi_institution(self):
             "northeastern.edu": "Northeastern University",
         }
 
-        self.assertEqual(
-            process_report.get_institution_from_pi(institute_map, "quanmp@bu.edu"),
-            "Boston University",
-        )
-        self.assertEqual(
-            process_report.get_institution_from_pi(
-                institute_map, "c@mclean.harvard.edu"
-            ),
-            "McLean Hospital",
-        )
-        self.assertEqual(
-            process_report.get_institution_from_pi(institute_map, "b@harvard.edu"),
-            "Harvard University",
-        )
-        self.assertEqual(
-            process_report.get_institution_from_pi(institute_map, "fake"), ""
-        )
-        self.assertEqual(
-            process_report.get_institution_from_pi(
-                institute_map, "pi@northeastern.edu"
-            ),
-            "Northeastern University",
-        )
+        answers = {
+            "q@bu.edu": "Boston University",
+            "c@mclean.harvard.edu": "McLean Hospital",
```

Comment:
would it be helpful to add these items to the test as well?
"childrens.harvard.edu" : "Boston Children's Hospital",
    "meei.harvard.edu"      : "Massachusetts Eye & Ear",
    "dfci.harvard.edu"      : "Dana-Farber Cancer Institute",
    "bwh.harvard.edu"       : "Brigham and Women's Hospital",
    "bidmc.harvard.edu"     : "Beth Israel Deaconess Medical Center",

Reply:
If we want to test more domains, I guess the question would be do we want to test for all the domains in `institute_map.json`, or just a few examples that validates the `get_institution_from_pi` function. @naved001 What is your opinion?

Reply:
@QuanMPhm I think it's okay to test the additional domains that @joachimweyl suggested.